test/server/schema_validate.spec.js

```diff
@@ -0,0 +1,237 @@
+/* globals describe, it, expect, beforeAll, afterAll*/
```

Comment:
i don't think beforeAll and afterAll are being used. So if you delete them from the rest of the code, they can also be deleted here.